students/roslyn_m/lesson08/test_circle.py

```diff
@@ -0,0 +1,143 @@
+import pytest
+from circle import *
```

Comment:
in general you want to avoid start imports, instead -> `from circle import Circle`